diff --git a/grader/settings.py b/grader/settings.py
index 9cfc947..eaab224 100644
--- a/grader/settings.py
+++ b/grader/settings.py
@@ -55,6 +55,7 @@ INSTALLED_APPS = (
     # 'django.contrib.contenttypes',
     # 'django.contrib.sessions',
     # 'django.contrib.messages',
+    'corsheaders',
     'staticfileserver', # override for runserver command, thus this needs to be before django contrib one
     'django.contrib.staticfiles',
     'access',
@@ -65,6 +66,7 @@ MIDDLEWARE = [
     # 'django.middleware.security.SecurityMiddleware',
     # 'django.contrib.sessions.middleware.SessionMiddleware',
     # 'django.middleware.csrf.CsrfViewMiddleware',
+    'corsheaders.middleware.CorsMiddleware',
     'django.middleware.clickjacking.XFrameOptionsMiddleware',
     # 'django.contrib.auth.middleware.AuthenticationMiddleware',
     # 'django.middleware.locale.LocaleMiddleware',
@@ -72,6 +74,8 @@ MIDDLEWARE = [
     'aplus_auth.auth.django.AuthenticationMiddleware',
 ]

+CORS_ALLOW_ALL_ORIGINS = True
+
 CACHED_LOADERS = [
     (
         'django.template.loaders.filesystem.Loader',
diff --git a/requirements.txt b/requirements.txt
index 4243873..5886328 100644
--- a/requirements.txt
+++ b/requirements.txt
@@ -6,3 +6,4 @@ docutils ~= 0.17.1
 python-magic ~= 0.4.27
 aplus-auth ~= 0.2.2
 docker ~= 7.1.0
+django-cors-headers ~= 3.13.0
